plugins/patriot-gcp/lib/patriot_gcp/command/load_to_bigquery.rb

```diff
@@ -4,13 +4,27 @@ class LoadToBigQueryCommand < Patriot::Command::Base
       declare_command_name :load_to_bigquery
       include PatriotGCP::Ext::BigQuery
 
-      command_attr :inifile, :dataset, :table, :schema, :options, :input_file, :name_suffix, :polling_interval
+      command_attr :inifile, :project_id, :dataset, :table, :schema, :options, :input_file, :name_suffix, :polling_interval
 
       class BigQueryException < Exception; end
       class GoogleCloudPlatformException < Exception; end
 
+      def set_conf
+        ini = IniFile.load(@inifile)
+        if ini.nil?
+          raise Exception, "inifile not found"
+        end
+
+        @service_account  = ini["gcp"]["service_account"]
+        @private_key      = ini["gcp"]["private_key"]
+        @key_pass         = ini["gcp"]["key_pass"]
+        @project_id     ||= ini["bigquery"]["project_id"]
+        @dataset        ||= ini["bigquery"]["dataset"]
+      end
+
       def job_id
-        job_id = "#{command_name}_#{@dataset}_#{@table}"
+        set_conf
```

Comment:
set_conf should be invoked in configure.
(e.g., https://github.com/CyberAgent/patriot-workflow-scheduler/blob/develop/lib/patriot/command/sh_command.rb#L36)